### ISSUE TITLE
why not stop gradient in mlora net code? 

### ISSUE THREAD

**Issue by LawrenceXia2008** [gaohaining/MLoRA#2]
why to commit the part of stop gradient of main backbone?

**Comment by bamboo-boy** [comment]
We believe that the backbone MLP has learned the information of all domain data in the pre-training stage, and only considers each domain special parameter when training each domain separately in the finetune stage. It is better to freeze the backbone MLP parameter.
